lib/segment/src/index/payload_config.rs

```diff
@@ -69,6 +69,19 @@ impl PayloadIndices {
         })
     }
 
+    /// Check if any payload field used RocksDB or could be using rocksDB.
+    /// This means that if we have no data about an indexes' type, we still return `true`.
```

Comment:
```suggestion
    /// This means that if we have no data about an index's type, we still return `true`.
```
I have no context of this PR. But nit :)